ElasticBox/partner-cloud-integration-azure-deleteCustomerAcount.md

```diff
@@ -0,0 +1,36 @@
+{{{
+  "title": "Partner Cloud Integration: Deleting Azure Customer Accounts",
+  "date": "01-16-2017",
+  "author": "Ben Swoboda",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+###Overview
+
+Partner Cloud Integration allows an ElasticBox user to create Azure Customer Accounts. If you want to stop usage and completely shut down that Azure Customer Account, you must do more than delete the provider in ElasticBox. You will also need to tell CenturyLink to turn off the actual Azure account.
+
+### Audience
+
+All ElasticBox Customers with ARM Providers supported by CenturyLink
+
+### Prerequisites
+
+You will need to have followed [the New ARM Provider process](../elasticbox/partner-cloud-integration-azure-new.md) at some point already.
```

Comment:
Change the link to `../ElasticBox/partner-cloud-integration-azure-new.md`. Case is significant. The folder name is `ElasticBox` not `elasticbox`. Change this in the rest of the places this occurs in this PR.